lago/virt.py

```diff
@@ -114,6 +115,51 @@ def _create_vm(self, vm_spec):
         vm_spec['vm-type'] = vm_type_name
         return vm_type(self, vm_spec)
 
+    def check_memory_requirements(self, conf):
```

Comment:
I think this is still relevant (at last to issue a a warning) even if only based on libvirt